pyproject.toml

```
@@ -57,6 +57,7 @@ generation_data = "gen"
 Packet = "photonlib/Packet.h"
 PhotonCamera = "photonlib/PhotonCamera.h"
 PhotonPipelineResult = "photonlib/PhotonPipelineResult.h"
+PhotonPoseEstimator = "photonlib/PhotonPoseEstimator.h"
```

per https://github.com/robotpy/robotpy-photonvision/pull/11#discussion_r1083219999
```suggestion
#PhotonPoseEstimator = "photonlib/PhotonPoseEstimator.h"
```